src/games/strategy/triplea/delegate/StrategicBombingRaidBattle.java

```diff
@@ -203,7 +203,21 @@ public void fight(final IDelegateBridge bridge) {
     showBattle(bridge);
     final List<IExecutable> steps = new ArrayList<>();
     if (hasAA) {
-      steps.add(new FireAA());
+    	boolean foundGenericAA = false;
```

Comment:
It's correct the way it is unless I'm missing something. Suppose that there are AA firing at everything and AA only firing at one thing in the same territory (unlikely a map would do that but let's say its possible). The generic AA would need to fire, and then the specific AA.

Basically the existing code was only correct for the case where the targets did not have the AAforBombingThisUnitOnly flag set to true. Global 1940 has it set to true in all cases I'm aware of.